documentation/release_notes.markdown

```diff
@@ -109,131 +162,150 @@ stable release of PuppetDB, which included an experimental v4 API.
     For API consumers this means, the extra `data` key needs to be traversed to access the real related
     object data.
 
-* Add "noop" flag to reports ([PDB-1177](https://tickets.puppetlabs.com/browse/PDB-1177))
+* Add `noop` flag to reports ([PDB-1177](https://tickets.puppetlabs.com/browse/PDB-1177))
 
-    This adds a "noop" field to the reports object, which is a
+    This adds a `noop` field to the reports object, which is a
     required boolean flag indicating whether the run producing the
-    report was a --noop.
+    report was using `--noop` (for example: `puppet agent -t --noop`).
 
-* Remove 'com.' prefix from JMX and ActiveMQ endpoints ([PDB-863](https://tickets.puppetlabs.com/browse/PDB-863))
+* Remove `com.` prefix from JMX and ActiveMQ endpoints ([PDB-863](https://tickets.puppetlabs.com/browse/PDB-863))
 
-    The 'com.' prefix has been removed from the JMX and ActiveMQ endpoints. They are now in the
-    puppetlabs.puppetdb namespace.
+    The `com.` prefix has been removed from the JMX and ActiveMQ endpoints. They are now in the
+    `puppetlabs.puppetdb` namespace. Any monitoring applications you have that
+    expect the old namespace will need to be changed, along with the general
+    URL change from `/v4/metrics` to `/pdb/metrics/v1`.
+
+* The endpoints `aggregate-event-counts` and `event-counts` have now been marked as experimental.
+
+    These endpoints were introduced to solve specific problems for the PE console,
+    however the design is under question. We're moving these endpoints back to
+    experimental status so that we are able to deliver a better option later on.
+
+* The older puppetdb-\* sub-commands are now retired (`puppetdb-ssl-setup` for example) instead,
+  the new sub-command style should be used `puppetdb ssl-setup`. ([PDB_663](https://tickets.puppetlabs.com/browse/PDB-663))
```

Comment:
We have puppetdb ssl-setup as the "right way" since way back to 1.6 but we hadn't removed/retired those other scripts until 3.0.0.